test.sh

```diff
@@ -137,7 +137,23 @@ gometalinter.v2 --disable-all \
     --enable=lll \
     --dupl-threshold=400 \
     --enable=dupl \
-    ./pkg/...
+    ./pkg/admission/ \
```

Comment:
Is there any `-execlude` flag so that you don't need to list all other directories?

Reply:
Yeah, found the `skip` flag.
Updated.